src/components/Match/matchColumns.jsx

```diff
@@ -462,15 +462,17 @@ export const fantasyColumns = [
   },
 ].concat(fantasyComponents);
 
-export const purchaseTimesColumns = (match) => {
+export const purchaseTimesColumns = (match, showCommItems) => {
+  // IDs of common consumable items
+  const commCon = showCommItems ? [] : [46, 216, 44, 42, 43, 38, 257, 265, 188, 16];
```

Comment:
I suggest using the names for greater readability

Reply:
shoul I put the array with item names outside the function?

Reply:
sure you can put it in utility file if you think it might be reused

Reply:
ok done